Automattic/wp-calypso
# One-click upsell A/B test for Personal to Premium upgrade

#### Changes proposed in this Pull Request

* Implement an A/B test for Personal plan purchases:
   - Control group will see a Quick Start offer after purchase completes for a Personal plan
    - Treatment group will see a offer for Premium plan upgrade with one-click purchase enabled.

* For more context, please read pcbrnV-LX-p2.

**Notes before you begin review**

This PR looks bigger than the actual code changes because I've moved some existing files to new folders, so those appear as new code changes. Here's a quick summary of some of those changes for easier review:

* Renamed the folder `client/my-sites/checkout/upsell-nudge/plan-upgrade-upsell` to `client/my-sites/checkout/upsell-nudge/business-plan-upgrade-upsell/`
* Created a new folder `client/my-sites/checkout/upsell-nudge/premium-plan-upgrade-upsell/` that has the Premium upsell offer page and its related sass file.
* Important: The copy for the Premium upgrade offer is right now a placeholder content and not the final version. It will be replaced with the actual copy whenever it's ready. 



#### Testing instructions



1. Assign yourself to the treatment of `one_click_premium_plan_upgrade` A/B test and to `offer` variation of `conciergeUpsellDial` test.
2. Signup for a new WPCOM account and purchase a Personal plan
3. Verify that you see the Premium plan upsell offer.
4. Start over but this time assign to control of `one_click_premium_plan_upgrade` A/B test. Repeat steps 1-3 and verify that you see  the Quick Start offer.
